Homework/Homework_5/001.py

```diff
@@ -0,0 +1,12 @@
+# Напишите программу, удаляющую из текста все слова, в которых присутствуют все буквы "абв".
+
+s = [i for i in str(input('Введите текст: ')).split()]
```

Comment:
```suggestion
s = input('Введите текст: ').split()
```
Так ведь проще. Не стоит усложнять.